### PR TITLE
AssetMapper 6.4 docs

### PR DESCRIPTION
**Pull request by weaverryan**
Hi!

A few notable things:

A) Of course, AssetMapper docs were updated for 6.4. Many features were already done - thank you for the community for doing that! 

B) I set AssetMapper as the recommended assets solution for 6.4.

C) I Added "Encore" to many of the Encore documentation pages for clarity. I also moved the Encore TOC to its own page. This makes the frontend page short and focused: a landing point to go elsewhere.

Cheers!

### CODE REVIEW

**Comment by weaverryan** [comment]
Ok, should be ready to go - read and proofread again. The Code Blocks failure isn't legit - it can't find the `importmap()` function.

**Comment by smnandre** [comment]
> Ok, should be ready to go - read and proofread again. The Code Blocks failure isn't legit - it can't find the `importmap()` function.

I think asset-mapper is not installed in the test application: https://github.com/symfony-tools/symfony-application/blob/6.4/composer.json

**Comment by weaverryan** [comment]
> I think asset-mapper is not installed in the test application: https://github.com/symfony-tools/symfony-application/blob/6.4/composer.json

Apparently I thought of that a long time ago, but forgot: https://github.com/symfony-tools/symfony-application/pull/16

It shouldn't be a blocker. But yes, we should merge that at some point.

File: frontend.rst [hunk 1]
```diff
@@ -8,34 +8,45 @@ more advanced - like scaffolding your front-end with a tool like Next.js.
...
-Supports `Stimulus/UX`_           yes                yes
-Supports Sass/Tailwind            yes                :ref:`yes <asset-mapper-tailwind>`
-Supports React, Vue, Svelte?      yes                yes :ref:`[1] <ux-note-1>`
-Supports TypeScript               yes                no :ref:`[1] <ux-note-1>`
-================================  =================  ======================================================
+* :ref:`AssetMapper <frontend-asset-mapper>` (recommended for new projects) runs
+  entirely in PHP, doesn't require any build step and leverages modern web standards.
+
+* :ref:`Webpack Encore <frontend-webpack-encore>` is built with `Node.js`_
+  on top of `Webpack`_ and.
```
Comment (javiereguiluz) [inline frontend.rst]: the `and.` at the end of the phrase looks strange

File: frontend/encore/split-chunks.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
-Preventing Duplication by "Splitting" Shared Code into Separate Files
-=====================================================================
+Encore: Preventing Duplication by "Splitting" Shared Code
```
Comment (javiereguiluz) [inline frontend/encore/split-chunks.rst]: Any reason for not applying here the same format as other updated headings?

```
Encore: Preventing Duplication by "Splitting" Shared Code
Preventing Duplication by "Splitting" Shared Code with Webpack Encore
```

**Review by javiereguiluz — APPROVED** [review]
I like this a lot. Let's promote AssetMapper as the default and recommended way. Thanks Ryan!

**Pull request merged**